src/teuthology_api/config.py

```diff
@@ -1,43 +0,0 @@
-from functools import lru_cache
-from pydantic_settings import BaseSettings, SettingsConfigDict
-from dotenv import load_dotenv
-
-load_dotenv()
-
-
-class APISettings(BaseSettings):
-    """
-    Class for API settings.
-    """
-
-    deployment: str = ""
-    pulpito_url: str = ""
-    paddles_url: str = ""
-
-    gh_client_id: str = ""
-    gh_client_secret: str = ""
-    gh_token_url: str = ""
-    gh_authorization_base_url: str = ""
-    gh_fetch_membership_url: str = ""
-
-    session_secret_key: str = ""
-    archive_dir: str = ""
-    teuthology_path: str = ""
-
-    model_config = SettingsConfigDict(
-        env_file=".env", env_file_encoding="utf-8", extra="ignore"
-    )
-    # TODO: team names need to be changed below when created
-    admin_team: str = "ceph"  # ceph's github team with *sudo* access to sepia
-    teuth_team: str = "teuth"  # ceph's github team with access to sepia
```

Comment:
@VallariAg should we be worried about `admin_team` and `teuth_team` variables? I know you had the admin privilege change PR, which is an auxiliary to your kill PR.